src/cljam/io/vcf/writer.clj

```diff
@@ -32,7 +32,7 @@
 
 (defn- nil->dot
   [s]
-  (if (nil? s) "." s))
+  (or s "."))
```

Comment:
By the rewrite, it will also convert `false` to dot (`"."`). I think it's a good thing to undo the code or change the function name as something like `default-dot-value`. It's only a nitpick.

Reply:
Though there's no concept of boolean values in the current version of VCF/BCF spec, I agree with you that it's misleading. Reverted in a455eac.